ansible/roles/operator-pipeline/tasks/webhook-hosted-pipeline-trigger.yml

```diff
@@ -81,7 +81,7 @@
                 metadata:
                   generateName: operator-hosted-pipeline-run
                 spec:
-                  timeout: "2h"
+                  timeout: "1.5h"
```

Comment:
This is changing the timeout on the wrong pipeline. Please add the 1.5h timeout to the release pipeline trigger instead.

Reply:
https://github.com/redhat-openshift-ecosystem/operator-pipelines/blob/main/operator-pipeline-images/tests/data/pipelinerun.json#L332

Are below changes correct ?
`"timeout": "1h0m0s", to "timeout": "1h30m0s",`

Reply:
It needs to be added here: https://github.com/redhat-openshift-ecosystem/operator-pipelines/blob/a9f8286bdf42ad4a01312f725cd55d1f7c10d114/ansible/roles/operator-pipeline/tasks/webhook-release-pipeline-trigger.yml#L65